sdk/messaging/azservicebus/internal/namespace.go

```diff
@@ -215,26 +211,21 @@ func (ns *Namespace) NewAMQPSession(ctx context.Context) (AMQPSessionCloser, uin
 	return session, clientRevision, err
 }
 
-// NewMgmtClient creates a new management client with the internally cached *amqp.Client.
```

Comment:
@jhendrixMSFT - no longer needed since we centered around using the rpcLink in amqpLinks, rather than using ManagementClient.